app/views/announcements/index.html.erb

```diff
@@ -2,18 +2,21 @@
 <div class = "sticky-top">
   <!-- General Nav bar -->
   <nav class="navbar navbar-expand-lg navbar-light bg-dark p-0">
-    <div class="container-fluid">
-      <a class="navbar-brand" href="#">
-        <%= image_tag "bsfc_logo.png", {:alt => "Berkeley Student Food Collective Logo", :width => 40, :height => 40} %>
+    <div class = "container-fluid">
```

Comment:
spaces around `=` signs are an antipattern for HTML.  it looks like these files were correct before but this error got introduced. pls clean up.